newm-server/src/main/kotlin/io/newm/server/features/arweave/repo/Util.kt

```diff
@@ -0,0 +1,83 @@
+package io.newm.server.features.arweave.repo
+
+import com.amazonaws.HttpMethod
+import com.amazonaws.services.s3.AmazonS3
+import io.ktor.server.application.*
+import io.newm.server.features.arweave.model.WeaveFile
+import io.newm.server.features.arweave.model.WeaveProps
+import io.newm.server.features.arweave.model.WeaveRequest
+import io.newm.server.features.song.model.Song
+import io.newm.server.ktx.asValidUrl
+import io.newm.server.ktx.getSecureConfigString
+import io.newm.server.ktx.toBucketAndKey
+import io.newm.shared.koin.inject
+import io.newm.shared.ktx.info
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.koin.core.parameter.parametersOf
+import org.slf4j.Logger
+import java.time.Instant
+import java.time.temporal.ChronoUnit
+import java.util.*
+
+object Util {
```

Comment:
Rather than creating a Util for all of these, I would prefer an extension functions to map a single Song to the files map. Everything else can stay at ArweaveRepositoryImpl. Something like:
 ```
 fun Song.toFiles(): List<String, String>
 ```
 That would take everything form line 36 to 60.  That can go in a file: `/features/arweave/ktx/SongExt.kt`
Then line 35 shoukld look like:
```
files = song.toFiles()
```